### DISCUSSION TITLE
How do i launch multiple instances of the same minecraft (different accounts)

### DISCUSSION THREAD

**Discussion by itsnotethann** [ORelio/Minecraft-Console-Client#1753] [Q&A]
^

**Comment by ORelio** [comment]
Launch MinecraftClient.exe several times.
You can have the same config and pass the username as first argument.
Example script **launch.bat** that you can create with Notepad and rename from `.txt` to `.bat`:
```
@echo off
cd %~dp0
start /min MinecraftClient.exe user1
start /min MinecraftClient.exe user2
start /min MinecraftClient.exe user3
```
Define user1, user2, user3, etc in a file called [**accounts.txt**](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-accounts.txt) set in config file:
https://github.com/ORelio/Minecraft-Console-Client/blob/f16e9e87dd82381514d5ea11300d88b4d47dfd25/MinecraftClient/Resources/config/MinecraftClient.ini#L36
```
user1,playerone@email.com,password1
user2,playertwo@email.com,password2
user3,playerthree@email.com,password3
```
